manual/ssl/README.md

```diff
@@ -178,15 +177,16 @@ SslContextBuilder builder = SslContextBuilder
   // only if you use client authentication
   .keyManager(new File("client.crt"), new File("client.key"));
 
-SSLOptions sslOptions = new NettySSLOptions(builder.build());
+SSLOptions sslOptions = new RemoteEndpointAwareNettySSLOptions(builder.build());
 
 Cluster cluster = Cluster.builder()
   .addContactPoint("127.0.0.1")
   .withSSL(sslOptions)
   .build();
 ```
 
-[SSLOptions]: http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/SSLOptions.html
-[JdkSSLOptions]: http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/JdkSSLOptions.html
-[NettySSLOptions]: http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/NettySSLOptions.html
-[NettyOptions]: http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/NettyOptions.html
+[RemoteEndpointAwareSSLOptions]:      http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/RemoteEndpointAwareSSLOptions.html
```

Comment:
All these urls should be updated to 3.2, but I guess we can do that later when releasing 3.2.0 (I bet many other manual pages will have to be updated as well).